components/purchase_modal/process_payment_setup.tsx

```diff
@@ -312,6 +333,33 @@ class ProcessPaymentSetup extends React.PureComponent<Props, State> {
             );
         case ProcessState.SUCCESS:
             return this.sucessPage();
+        case ProcessState.FAILED_COMPLIANCE_SCREEN:
+            return (
+                <IconMessage
+                    title={t(
```

Comment:
wondering if `t` should be used here instead of this.props.intl ?

Reply:
The `IconMessage` component supports two apis. The API being used asks the user provide only an id, and the IconMessage component renders it as a FormattedMessage (without providing a defaultMessage). This is how a lot of the existing usage is.

There is also a formatted<PropName> api, that requests the user pass in an already internationalized ReactNode. In general I think the component should move towards removing all usages of the id-only API because of the pitfall of not providing a defaultMessage: languages not providing the translation string will make errors.

That said, there are other existing usages of the id-only api so I'm 0/5 on whether we want to make this usage use the formatted* API in this PR.

Reply:
Yup no worries at all just wanted to double check its usage